svg-spritemap.js

```diff
@@ -14,28 +16,74 @@ function SVGSpritemapPlugin(options) {
         glob: {},
         prefix: 'sprite-',
         gutter: 2,
-        filename: 'spritemap.svg'
+        filename: 'spritemap.svg',
```

Comment:
Better use output, instead of filename. This isn't clear where filename is used for.